.gitignore

```diff
@@ -39,7 +39,6 @@ faces-config.NavData
 .gradle
 .worktree
 work/
-public/
```

Comment:
This should not be removed?

Reply:
I need too add the new html file in a directory that is called public. Maybe I can add the file bypassing the gitignore then?

Reply:
maybe it's ok if nothing else is getting committed by removing this line